components/friends.js

```diff
@@ -23,18 +29,18 @@ function Friends( { route, data, isDarkTheme } ) {
   return (
     <TouchableWithoutFeedback onPress={() => {Keyboard.dismiss()}}>
       <ScrollView>
-      <SearchBarFriends searchFriend={searchFriend} userData={userData}/>
-            <View  style={styles.main}>{userData.map((e) => {
+      <SearchBarFriends searchFriend={searchFriend} listofusers={listofusers}/>
+            <View  style={styles.main}>{friendsList.map((e) => {
```

Comment:
When trying to run this, opening the Friends page results in an error of not being able to call `map` on `friendsList`. I assume this is because `friendsList` is not yet populated with data, but we'll still need to account for the possibility of an empty `friendsList` as that will be the case for everyone when they first sign up.